plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -28,13 +28,15 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/quota"
 	"k8s.io/kubernetes/pkg/quota/install"
+	utilruntime "k8s.io/kubernetes/pkg/util/runtime"
 )
 
 func init() {
 	admission.RegisterPlugin("ResourceQuota",
 		func(client clientset.Interface, config io.Reader) (admission.Interface, error) {
 			registry := install.NewRegistry(client)
-			return NewResourceQuota(client, registry, 5)
+			// TODO: expose a stop channel in admission factory
+			return NewResourceQuota(client, registry, 5, make(chan struct{}))
```

Comment:
no issue for `Wants` pattern, but I think either @aveshagarwal @sjenning will be working on it soon.